### ISSUE TITLE
"max_size" keyword raise error

### ISSUE THREAD

**Issue by GCer-Hidenori** [GCer-Hidenori/ANTSQLParser#52]
- input
```
SELECT max_size  
FROM a
```

- input
```
USE AdventureWorks2012;  
GO  
SELECT file_id, name, type_desc, physical_name, size, max_size  
FROM sys.database_files ;  
GO  
  

-- Database design/Databases/How-to guides/Manage/Display Data & Log Space Information for a Database
-- https://docs.microsoft.com/en-us/sql/relational-databases/databases/display-data-and-log-space-information-for-a-database?view=sql-server-ver15
```

**Comment by GCer-Hidenori** [comment]
- input
```
 SELECT name AS FileName,
    size*1.0/128 AS FileSizeInMB,
    CASE max_size
        WHEN 0 THEN 'Autogrowth is off.'
        WHEN -1 THEN 'Autogrowth is on.'
        ELSE 'Log file grows to a maximum size of 2 TB.'
    END,
    growth AS 'GrowthValue',
    'GrowthIncrement' =
        CASE
            WHEN growth = 0 THEN 'Size is fixed.'
            WHEN growth > 0 AND is_percent_growth = 0
                THEN 'Growth value is in 8-KB pages.'
            ELSE 'Growth value is a percentage.'
        END
FROM tempdb.sys.database_files;
GO

-- Database design/Databases/Concepts/System databases/tempdb
-- https://docs.microsoft.com/en-us/sql/relational-databases/databases/tempdb-database?view=sql-server-ver15
```

**Comment by GCer-Hidenori** [comment]
fix with #26
